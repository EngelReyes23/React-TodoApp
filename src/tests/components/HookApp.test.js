import HookApp from "../../HookApp";
import shallow from "enzyme";

describe("Probando HooApp", () => {
	// wrapper
	let wrapper = shallow(<HookApp />);

	beforeEach(() => {
		wrapper = shallow(<HookApp />);
	});

	test("debe mostrarse correctamente", () => {
		expect(wrapper).toMatchSnapshot();
	});
});
