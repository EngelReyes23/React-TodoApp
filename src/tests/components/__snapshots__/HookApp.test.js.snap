// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando HooApp debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Hook App
  </h1>
</div>
`;
